### DISCUSSION TITLE
Bug in satellites plugin?

### DISCUSSION THREAD

**Discussion by auj68** [Stellarium/stellarium#3155] [Q&A]
I'm a beginner with Stellarium and not sure if I'm doing something wrong or have I found a bug in the satellites plugin. To me it seems that observer's location affects the equatorial coordinates of a satellite (below a Starlink-1110 as an example). I have tested Stellarium versions 23.1 and 1.2, and they both give the same result. Stellarium 23.1 tested both on Win10 and Ubuntu 22.04.

Observer's latitude +90 (North Pole), equirectangular projection, equatorial grid in blue:
![image](https://user-images.githubusercontent.com/85354109/229356210-0d3e895f-076f-45fe-bd04-5fa9fd3c2224.png)
The orbit of Starlink-1110 oscillates between declination -12 and -70.

Observer's latitude -90 (South Pole), equirectangular projection is flipped upside down, north at the bottom:
![image](https://user-images.githubusercontent.com/85354109/229356720-b23a0461-b9cc-4457-89e5-9db42b3439fb.png)
The orbit of Starlink-1110 oscillates between declination +12 and +70.

The subpoint declination of Starlink-1110 varies between +53 and -53, which is correct with inclination +53.

Could you confirm if this is a bug? Thanks, Ari.

**Comment by gzotti** [comment]
Why should that be a bug? 
You are on the North or South pole of an extended body and observe another object in a close orbit. If you step aside by one metre, the satellite will shift by a tiny angle (parallax). If you move by 12756km, you can expect that the satellite will stand in front of very different stars. Stop time and move slowly from pole to pole to see a gradual shift.

**Reply by auj68** [reply]
My mistake, indeed. Equatorial coordinate system is often described as a geocentric system with reference equatorial plane going through the center of Earth. This works perfectly for distant stars and is a rather good approximation for the Moon. This is the first time I was working with close orbit objects, for which one has to parallel shift the reference plane to the location of the observer. The parallax effect indeed makes many satellite orbits look weird and unphysical if the equatorial coordinates are interpreted in a wrong geocentric way.

Thanks for pushing me into the right orbit.

**Reply by gzotti** [reply]
Stellarium has a switch to use (default) topocentric or geocentric coordinates. You could report that the Satellites plugin does not observe this setting and always shows topographic conditions. But the original developer is no longer available, and fixing that in code not developed by ourselves looks more difficult than useful, as nobody can observe from the geocenter.
